hackathon_site/registration/test_forms.py

```diff
@@ -84,18 +84,26 @@ class ApplicationFormTestCase(SetupUserMixin, TestCase):
     def setUp(self):
         super().setUp()
         self.data = {
```

Comment:
usually this object is filled only with required form data, so we make sure that the optional fields are not accidentally required fields and then we have a separate unit test just for the optional fields. So could you remove the optional fields (the social media links) from this object and add another test that only tests the optional fields?

Reply:
For the optional fields, what exactly should I be testing? @YuYing-Liang

Reply:
you basically add all the optional fields and build the form and it should succeed. Look back at newhacks 2021 for an example